plugins/receiver/grpc/nativelog/receiver.go

```diff
@@ -0,0 +1,66 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package nativelog
+
+import (
+	"fmt"
+	"reflect"
+
+	"google.golang.org/grpc"
+
+	logging "skywalking/network/logging/v3"
+
+	"github.com/apache/skywalking-satellite/internal/pkg/config"
+	"github.com/apache/skywalking-satellite/internal/pkg/plugin"
+	grpcreceiver "github.com/apache/skywalking-satellite/plugins/receiver/grpc"
+	"github.com/apache/skywalking-satellite/protocol/gen-codes/satellite/protocol"
+)
+
+type Receiver struct {
+	config.CommonFields
+	grpcreceiver.CommonGRPCReceiverFields
+	service *LogReportService // The gRPC request handler for logData.
+}
+
+func (r *Receiver) Name() string {
+	return plugin.GetPluginName(r)
```

Comment:
Ditto.
I think `GetPluginName` is not necessary.  And this forces us to use reflection on the interface to get the implement name.
We can make rules that  setting `Name` in each plugin instance directly.